_config.php

```diff
@@ -12,7 +16,7 @@
 	'Security/logout' => 'RestrictedSecurityController',
 ));
 
-Object::add_extension('Member', 'RestrictedMember');
+SS_Object::add_extension('Member', 'RestrictedMember');
```

Comment:
Can this just be replaced with `Member::add_extension()` to avoid the need for the class_alias?

Reply:
(can't recall whether 3.x supports looking up the called class for these things)

Reply:
Sure, done.